src/test/kotlin/com/buildit/bookit/v1/booking/BookingControllerMockMvcTests.kt

```diff
@@ -19,159 +19,131 @@ import org.junit.jupiter.api.extension.ExtendWith
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
 import org.springframework.boot.test.mock.mockito.MockBean
-import org.springframework.http.MediaType
+import org.springframework.http.MediaType.APPLICATION_JSON
 import org.springframework.test.context.junit.jupiter.SpringExtension
 import org.springframework.test.web.servlet.MockMvc
-import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
-import org.springframework.test.web.servlet.result.MockMvcResultMatchers
-import java.time.LocalDateTime
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import java.time.LocalDateTime.now
 import java.time.ZoneId
-import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME
 
 /**
- * Booking Contoller spring mvc integration tests
+ * Booking controller spring mvc integration tests
  */
 @ExtendWith(SpringExtension::class)
 @WebMvcTest(BookingController::class)
 class BookingControllerMockMvcTests @Autowired constructor(
-    private val mockMvc: MockMvc,
+    private val mvc: MockMvc,
     private val mapper: ObjectMapper
 ) {
+    private val NYC = "America/New_York"
+    private val NYC_TZ = ZoneId.of(NYC)
```

Comment:
if you really wanted to learn re: static fields https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#static-fields

Reply:
Heh.  That site does not even begin to layout in Safari.  I did try, but not so hard as to open in a different browser :-D.  Looks like I'll be applying `const1` in the next round.